src/User.js

```diff
@@ -7,38 +7,64 @@ class User {
     this.recipesToCook = [];
   }
 
-  updateFavorites(recipe) {
-    if (!this.favRecipes.includes(recipe)) {
-      this.favRecipes.push(recipe)
-    } else {
-      let recipeIndex = this.favRecipes.indexOf(recipe);
-      this.favRecipes.splice(recipeIndex);
-    }
-  }
+  // updateFavorites(recipe) {
+  //   if (!this.favRecipes.includes(recipe)) {
+  //     this.favRecipes.push(recipe)
+  //   } else {
+  //     let recipeIndex = this.favRecipes.indexOf(recipe);
+  //     this.favRecipes.splice(recipeIndex);
+  //   }
+  // }
+
+  // updateRecipesToCook(recipe) {
+  //   if (!this.recipesToCook.includes(recipe)) {
+  //     this.recipesToCook.push(recipe)
+  //   } else {
+  //     let recipeIndex = this.recipesToCook.indexOf(recipe);
+  //     this.recipesToCook.splice(recipeIndex);
+  //   }
+  // }
 
-  updateRecipesToCook(recipe) {
-    if (!this.recipesToCook.includes(recipe)) {
-      this.recipesToCook.push(recipe)
+  updateSavedRecipes(savedList, recipe) {
+    if (!savedList.includes(recipe)) {
+      savedList.push(recipe)
     } else {
-      let recipeIndex = this.recipesToCook.indexOf(recipe);
-      this.recipesToCook.splice(recipeIndex);
+      let recipeIndex = savedList.indexOf(recipe);
+      savedList.splice(recipeIndex);
     }
   }
```

Comment:
This was a really great idea! Awesome work, this way we have SRP methods for our user class.